### PR TITLE
Add .bash_profile dotfile to output of ls -F -a, and explanation thereof.

### PR DESCRIPTION
**Pull request by lexnederbragt**
Adresses #41.

### CODE REVIEW

File: 01-filedir.md [hunk 1]
```diff
@@ -336,7 +336,10 @@ As you can see,
 it also displays another special directory that's just called `.`,
 which means "the current working directory".
 It may seem redundant to have a name for it,
-but we'll see some uses for it soon.
+but we'll see some uses for it soon.  
+Finally, we now see a file called `.bash_profile`. This file usually contains settings to customize the shell (terminal). It may also be called `.bashrc` or `.bash_login`. For this lesson material it does not contain any settings.
```
Comment (ChristinaLK) [inline 01-filedir.md]: language pedantry: can you change "now" to something else, like "Finally, we also see..." or something along those lines?  "now" to me implies immediacy, and this is the last thing being discussed after the command/output is given.

**Comment by wking** [comment]
On Wed, Feb 04, 2015 at 11:40:41PM -0800, Lex Nederbragt wrote:
>     A filesystem/users/nelle/.bash_profile (3)

I like this example, and most of the content in .bash_profile.
However, I'd replace “It may also be called .bashrc or .bash_login”
with “There may also be similar files called .bashrc or .bash_login”,
because the three are not quite synonymous.  From the INVOCATION
section of bash(1):

  When bash is invoked as an interactive login shell, or as a
  non-inter‐ active shell with the --login option, it first reads and
  executes com‐ mands from the file /etc/profile, if that file exists.
  After reading that file, it looks for ~/.bash_profile,
  ~/.bash_login, and ~/.profile, in that order, and reads and executes
  commands from the first one that exists and is readable.  The
  --noprofile option may be used when the shell is started to inhibit
  this behavior.

  …

  When an interactive shell that is not a login shell is started, bash
  reads and executes commands from ~/.bashrc, if that file exists.
  This may be inhibited by using the --norc option.  The --rcfile file
  option will force bash to read and execute commands from file
  instead of ~/.bashrc.

And from the FILES section:

  ~/.bash_profile
    The personal initialization file, executed for login shells
  ~/.bashrc
    The individual per-interactive-shell startup file

So .bash_profile and .bash_login are roughly synonymous, with
.bash_login being a fallback for .bash_profile, but .bashrc is
executed in different (non-login) circumstances.  In fact, since your
example lacks something like:

  [[ -f ~/.bashrc ]] && . ~/.bashrc

nelle's .bashrc won't be sourced at all when invoking a login shell.

Anyhow, new users will certainly not care about all of that, but I'd
still prefer my vague wording to avoid giving the wrong impression.

File: 01-filedir.md (hunk 1 above)
Comment (lexnederbragt) [inline 01-filedir.md]: Thanks, adjusted accordingly.

**Comment by lexnederbragt** [comment]
Thanks, this I didn't know. Adjusted the text accordingly.

**Comment by wking** [comment]
With 0bef8e0 (Addresses language comments to first commit), this looks
good to me.

File: 01-filedir.md [hunk 2]
```diff
@@ -336,7 +336,10 @@ As you can see,
 it also displays another special directory that's just called `.`,
 which means "the current working directory".
 It may seem redundant to have a name for it,
-but we'll see some uses for it soon.
+but we'll see some uses for it soon.  
+Finally, we also see a file called `.bash_profile`. This file usually contains settings to customize the shell (terminal). There may also be similar files called `.bashrc` or `.bash_login`. For this lesson material it does not contain any settings.
```
Comment (willingc) [inline 01-filedir.md]: Looks good. I suggest adding to: For this lesson material the dotfile, `.bash_profile`, does not contain any workshop settings. The dotfile, `.bash_profile` would be hidden when executing an `ls` command without the `-a` "show all" option.

Comment (lexnederbragt) [inline 01-filedir.md]: Looks good. I suggest adding to: For this lesson material the dotfile, .bash_profile, does not contain any workshop settings.

I am afraid 'workshop settings' may be confusing...

The dotfile, .bash_profile would be hidden when executing an ls command without the -a "show all" option.

The whole point of adding is to show that adding '-a' to 'ls' shows this file. I feel adding this sentence is redundant and unnecessary.

Comment (willingc) [inline 01-filedir.md]: I agree with your point on 'workshop settings'.

I do think that it is helpful to 'new' and 'novice' users to explicitly state that without the '-a' option the dotfile will not be displayed. Although an experienced user may find it redundant, many new learners will need the cognitive reinforcement.

Comment (ChristinaLK) [inline 01-filedir.md]: I agree with @lexnederbragt here, the earlier paragraph explicitly states that `-a` reveals files with a leading dot, so I'm fine with what you had.

**Pull request merged**